### PR TITLE
changed instructions in the email body 

### LINKED ISSUE #3606: [Multiple-use] Add explanation to all node-based email notifications to reply-by-email
We have a new reply-by-email system! It's great, and means you can reply to many emails you get from PublicLab.org and expect your email to be added as a comment. 💬 💥 ✉️ 

However, not all our email notifications make this clear yet -- since this is new, most warn people NOT to reply by email. Let's change that! 🎉 

Changing it will take 3 steps per email template:

## 1. Change email subject lines to include node ids in format `(#0000)`

The new feature works by looking for a node ID in the subject line, looking up that node and attaching response emails to that node as comments. So we need each email notification to have a specially formatted node id in the subject line. Here's how they should change:

```diff
    mail(to: user.email, subject: 'You were mentioned in a comment.')
    mail(to: user.email, subject: "You were mentioned in a comment. (##{comment.node.id}) ")
```

The variable may be `node.id` instead of `comment.node.id` -- it depends on the context. You can find all the mailer functions in this folder, so look through to see which one you want to take on:

https://github.com/publiclab/plots2/tree/master/app/mailers

## 2. Change instructions in email body

Many email addresses warn you not to reply by email with the phrase `Do NOT reply to this email`. You can find them in this search (until we get them all!):

https://github.com/publiclab/plots2/search?q=%22Do+NOT+reply+to+this+email%22&unscoped_q=%22Do+NOT+reply+to+this+email%22

Once the email subject line is changed for a given template (and not before!) we can change this instruction from the top line to the bottom line:

```diff
- Hi! There's been a response to a discussion you're involved in. Do NOT reply to this email; click this link to respond:
+ Hi! There's been a response to a discussion you're involved in. You can reply to this email or visit this link:
```

See how someone did this for an existing template here: https://github.com/publiclab/plots2/commit/990c965aea47da7613c2825949a42f1ca7c29ec1

## 3. Update test

We'll help you with this step. Basically we have tests to confirm the subject line of each message sent. We'll need to update the tests to reflect the new subject lines, like this!

```diff
-    assert_equal "[PublicLab] #{user.username} liked your research note", email.subject
+   assert_equal "[PublicLab] #{user.username} liked your research note (##{node.id})", email.subject
```

That's it! Please take on only one email notification at a time in a pull request, and we can help and double-check your work. 

Also, some email notifications are not linked to just a single node -- so those won't be helpful to convert. If in doubt, link to the one you want to try (in the comments below) and we'll explain! 👍

### PR DESCRIPTION
**Pull request by rohan2599**
Fixes #3606 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

File: app/views/map/_maps.html.erb [hunk 1]
```diff
@@ -20,7 +20,6 @@
   <hr class="visible-xs visible-sm" />
 <% if ((i+1)/4.0).to_i == ((i+1)/4.0) %>
 </div>
-<hr class="visible-xs visible-sm grid" />
```
Comment (gauravano) [inline app/views/map/_maps.html.erb]: Please exclude this line as it's not mentioned in the issue. Thanks!

**Comment by rohan2599** [comment]
@gauravano please review I have removed the file

**Comment by rohan2599** [comment]
@jywarren @SidharthBansal  please review again. I have done the changes
thanks

**Comment by jywarren** [comment]
Ah, this completely solves all of that issue? Thank you! 

We do sometimes mark things as multiple use to give more people a chance to be involved - but we appreciate your work on this!

**Pull request merged**